pallets/reserve/src/lib.rs

```diff
@@ -20,6 +20,7 @@
 
 //! A module that is called by the `collective` and is in charge of holding
 //! the company funds.
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
I do not think this necessary as it is already at the top of `benchmarking.rs`?

Reply:
I will double check